### ISSUE TITLE
tornado < 4 consistently makes requests to luigid fail with 500 Internal server error

### ISSUE THREAD

**Issue by nresare** [spotify/luigi#1394]
Using luigid with a version earlier than 4.0 makes a simple hello world fail with a `KeyError: 'worker'`and the stack trace below.

Steps to reproduce:
*  in a virtualenv pip install luigi, then force-downgrade tornado: `pip install tornado==3.1.1`
* start a luigid in one window
* run a trivial luigi Task in another.

Suggested solution: enforce at least tornado 4.0
```
2015-11-12 15:11:32,176 tornado.application[79210] ERROR: Uncaught exception GET /api/add_task (10.48.20.57)
HTTPRequest(protocol='http', host='lon6-luigid-edgedocker1:8082', method='GET', uri='/api/add_task', version='HTTP/1.1', remote_ip='10.48.20.57', headers={'Content-Length': '477', 'Accept-Encoding': 'gzip, deflate, compress', 'Content-Type': 'application/x-www-form-urlencoded', 'Host': 'lon6-luigid-edgedocker1:8082', 'Accept': '*/*', 'User-Agent': 'python-requests/2.2.1 CPython/2.7.6 Linux/4.1.10-boot2docker'})
Traceback (most recent call last):
  File "/home/noa/luigid-test/local/lib/python2.7/site-packages/tornado/web.py", line 1141, in _when_complete
    callback()
  File "/home/noa/luigid-test/local/lib/python2.7/site-packages/tornado/web.py", line 1162, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/home/noa/luigid-test/local/lib/python2.7/site-packages/luigi/server.py", line 79, in get
    result = getattr(self._scheduler, method)(**arguments)
  File "/home/noa/luigid-test/local/lib/python2.7/site-packages/luigi/scheduler.py", line 605, in add_task
    worker_id = kwargs['worker']
KeyError: 'worker'
2015-11-12 15:11:32,177 tornado.access[79210] ERROR: 500 GET /api/add_task (10.48.20.57) 1.21ms
```

**Comment by nresare** [comment]
Suggested solution: https://github.com/spotify/luigi/pull/1395

**Comment by erikbern** [comment]
Is there some trivial way to fix the issue for older versions of Tornado?

**Comment by nresare** [comment]
@erikbern I have not looked into it. I guess my opinion on that is that if someone needs that functionality they are welcome to provide a patch, for now it makes sense to avoid the bad failure mode ;) Is that too harsh?

**Comment by erikbern** [comment]
sounds good to me!

**Comment by Tarrasch** [comment]
Fixed in #1395. I *think* this was introduced as of #1129. I can't be bothered to verify it though ^^
